sunpy/coordinates/frames.py

```diff
@@ -214,6 +225,13 @@ class HeliographicCarrington(HeliographicStonyhurst):
     >>> sc
     <SkyCoord (HeliographicCarrington: obstime=2010-01-01T00:00:45.000): (lon, lat, radius) in (deg, deg, km)
         [(1., 4., 5.), (2., 5., 6.), (3., 6., 7.)]>
+
+    >>> sc = SkyCoord(CylindricalRepresentation(0*u.km, 45*u.deg, 2*u.km),
```

Comment:
This is an odd choice of example because `CylindricalRepresentation` is uncommon, and even more so with a radius of zero (because then the angle value is degenerate).  I suggest writing a different example.